src/Project/Habitat/code/styles/Habitat/CoveoHabitat.css

```diff
@@ -0,0 +1,257 @@
+﻿/*Page in general*/
```

Comment:
Ajoute des spaces au début et à la fin de ton commentaire.

Ça serait bien de splitter les customization "Habitat-only" dans un autre fichier.

`.page-icon`, `main > .section:first-child, main > section:first-child` and such.